react-scanner.config.js

```diff
@@ -0,0 +1,34 @@
+module.exports = {
+	// Crawl the entire repo
+	crawlFrom: './',
+	// Needed for properly reporting components with dot notation
+	includeSubComponents: true,
+	// Exclude usage in tests, stories, and React Native files.
+	globs: [ '**/!(test|stories)/!(*native).@(js|ts)?(x)' ],
+	// Exclude any vendor or docs directories
+	exclude: [
+		'bin',
+		'build',
+		'build-module',
+		'docs',
+		'node_modules',
+		'patches',
+		'platform-docs',
+		'test',
+		'tools',
+		'typings',
+		'vendor',
+	],
+	/*
+	 * Filter out any non-component React elements and consider only imports of
+	 * `@wordpress/components` outside of the package.
+	 */
+	importedFrom: '@wordpress/components',
+	// Merge experimental and unstable stats into the canonical component name
+	getComponentName: ( { imported, local } ) => {
+		return ( imported || local )
+			.replace( '__experimental', '' )
+			.replace( '__unstable', '' );
+	},
```

Comment:
Should we keep this info separate so that we can track stabilization of an API, hoping to see a reduction of the prefixed APIs and an increase of the stable? We could aggregate at the "data analysis" / visualization level?

Reply:
I think you have a good point. Removed the canonization in ef0407efd0342d670ee24495ea4adcc13e85540e. We can tweak in the future if necessary.